### ISSUE TITLE
SubscriptionResolver is not used in Subscription-related fields

### ISSUE THREAD

**Issue by kamilkisiela** [dotansimha/graphql-code-generator#727]
**Describe the bug**
SubscriptionResolver is not used in Subscription-related fields

**To Reproduce**
```graphql
type Subscription {
  message: Message
}

schema {
  subscription: Subscription
}
```

```ts
import { SubscriptionResolvers } from './generated';

// SubscriptionResolvers.MessageResolver should equal SubscriptionResolver
```

**Expected behavior**
```ts
import { SubscriptionResolvers } from './generated';

// SubscriptionResolvers.MessageResolver equals Resolver
```
**Environment:**
 - OS: X
 - Codegen: 0.12.6
 - Node: JS

**Comment by kamilkisiela** [comment]
@dotansimha xD

https://github.com/dotansimha/graphql-code-generator/blob/25f355caceed2e7da15647b38a44886cef306e4e/packages/templates/typescript/tests/resolvers.spec.ts#L207-L215

**Comment by dotansimha** [comment]
@moonray fixed in 0.13.0.
